src/user/mod.rs

```diff
@@ -419,54 +391,3 @@ pub fn updates(req: &mut Request) -> CargoResult<Response> {
         meta: Meta { more: more },
     }))
 }
-
-#[cfg(test)]
```

Comment:
Looking at the [commit which added them](https://github.com/rust-lang/crates.io/commit/9ed51dd2) they seem to be testing something very specific which is still relevant here